components/ai_chat/resources/page/components/skill_modal/skill_modal.tsx

```diff
@@ -192,7 +197,11 @@ export default function SkillModal() {
                 className={classnames({
                   [styles.gradientIcon]: selectedModel === AUTOMATIC_MODEL_KEY,
                 })}
-                name={getModelIcon(selectedModel)}
+                name={
+                  selectedModelObject
+                    ? getModelIcon(selectedModelObject)
+                    : 'product-brave-leo'
+                }
```

Comment:
Why do we need to rule out selected models? 
If the reason for that is valid then don't just hardcoded `product-brave-leo` which would make future maintenance harder, use `getModelIcon('chat-automatic')` instead.